spec/models/instrument_spec.rb

```diff
@@ -718,7 +718,7 @@
         context "but it was canceled" do
           let(:user) { FactoryGirl.build :user }
           before :each do
-            Timecop.travel(60.minutes.ago) do
+            travel(-60.minutes) do
```

Comment:
I don't mind `travel(x.minutes)` to `travel_to(x.minutes.from_now)`, but I kind of prefer `travel_to(60.minutes.ago)` to using negative minutes. What do you think?